html/docs/include/functions/xdebug_get_gcstats_filename

```diff
@@ -0,0 +1,8 @@
+= xdebug_get_gcstats_filename
```

Comment:
Obscure feature: add ``,2.6`` at the end of the function name to show in which version this is introduced.